lib/mongo/crypt/context.rb

```diff
@@ -88,7 +88,7 @@ def run_state_machine
             filter = Binding.ctx_mongo_op(self)
 
             result = @encryption_io.collection_info(filter)
-            mongocrypt_feed(result)
+            mongocrypt_feed(result||{})
```

Comment:
This appears to not match the LMC integration guide - if result is nil don't feed it at all.

Reply:
I added a comment referencing a follow-up ticket.